.ci/pytorch/check_binary.sh

```diff
@@ -216,6 +216,14 @@ else
   fi
 fi
 
+###############################################################################
+# Check XPU configured correctly
+###############################################################################
+if [[ "$DESIRED_CUDA" == 'xpu' && "$PACKAGE_TYPE" != 'libtorch' ]]; then
+  echo "Checking that xpu is compiled"
+  python -c 'import torch; exit(0 if torch.xpu._is_compiled() else 1)'
```

Comment:
ditto

Reply:
I'd like to keep the method in here, due to it align with others check in this shell script